e2e/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "opentelemetry-e2e-test",
+    "private": true,
+    "description": "End-to-end tests for OpenTelemetry JS",
+    "version": "0.0.0",
+    "dependencies": {
+        "@opentelemetry/api": "*",
+        "@opentelemetry/api-logs": "*",
+        "@opentelemetry/sdk-node": "*",
+        "@opentelemetry/exporter-trace-otlp-proto": "*",
+        "@opentelemetry/exporter-metrics-otlp-proto": "*",
+        "@opentelemetry/exporter-logs-otlp-proto": "*",
+        "@opentelemetry/sdk-metrics": "*",
+        "@opentelemetry/sdk-logs": "*",
+        "@opentelemetry/sdk-trace-base": "*"
+    }
```

Comment:
nit: we actually used the `@opentelemetry/` prefix for all of our packages and linked explicit versions before. It's not too important to me though.

Reply:
I left it like this because its unpublished and I wanted to make sure no matter what we were always getting the local repo version. I think this makes it clear that we're not pulling specific versions of anything.